pkg/template/cue.go

```diff
@@ -63,7 +57,7 @@ func (c *cue) template(dirPath string, input io.Reader) exec.CmdRunResult {
 		args = append(args, c.opts.Paths...)
 	}
 
-	vals := Values{c.opts.ValuesFrom, c.appContext, c.coreClient, c.deployFactory, c.log}
+	vals := Values{c.opts.ValuesFrom, c.appContext, nil, nil}
```

Comment:
gotta revisit this, maybe moving the values into their own factory as suggested by DK